src/components/Contact/contact.css

```diff
@@ -0,0 +1,51 @@
+.contact-form {
+  margin: 0 auto;
+}
+
+.form-input input[type='text'] {
+  margin-bottom: 30px;
+}
+
+.form-input input[type='text'],
+.form-textarea textarea {
+  width: 100%;
+  padding: 10px 20px;
+  background-color: var(--primary-element-color);
+  border-radius: 20px;
+  font-size: 1em;
+  font-family: inherit;
+}
+
+.form-input input[type='text']::placeholder,
+.form-textarea textarea::placeholder {
+  color: var(--secondary-text-color);
+}
+
+.submit-btn button {
+  width: 100%;
+  margin-top: 30px;
+  background-color: var(--accent-element-color);
+  color: var(--secondary-text-color);
+  border-radius: 20px;
+}
+
+[data-theme='dark'] textarea,
+input {
+  color: #ffffff;
```

Comment:
C'mon! color is not a variable, and usage in css selector but not in theme.

Reply:
my bad, fixed